libsql_test.go

```diff
@@ -519,6 +520,7 @@ func TestEncryption(tt *testing.T) {
 	t.FatalOnError(err)
 	db := sql.OpenDB(connector)
 	ctx, cancel := context.WithTimeout(context.Background(), 15*time.Second)
+	defer cancel()
 	_, err = db.ExecContext(ctx, "CREATE TABLE "+table+" (id INTEGER PRIMARY KEY, name TEXT)")
 	if err != nil {
 		cancel()
```

Comment:
You missed some places were cancel is called.

Reply:
yeap, removed them as well.